.travis.yml

```diff
@@ -10,9 +10,11 @@ install:
   - go get -v -d -t github.com/golang/protobuf/...
   - curl -L https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip -o /tmp/protoc.zip
   - unzip /tmp/protoc.zip -d $HOME/protoc
+  - go get -u honnef.co/go/tools/cmd/megacheck
```

Comment:
Revert. This will require a larger discussion to add a static analysis tool as part of the testing process.

Reply:
Done.